fix issue #248

kafaa-dev opened the pull request:
Issue: #284

Sebelum :
![image](https://user-images.githubusercontent.com/73120669/201028359-5f92a0e8-f9d0-41c6-b374-a86006c65351.png)

Sesudah :
![image](https://user-images.githubusercontent.com/73120669/201028412-b534ea52-b2c4-48ad-801d-858f08cd6fe1.png)

kafaa-dev committed:
fix: bug input username memanjang

cek issue #284

vitorizkiimanda reviewed (changes requested):
please remove console.log

vitorizkiimanda commented on resources/js/Components/Dashboard/CreatePost.jsx:
```diff
@@ -15,6 +15,8 @@ const CreatePost = ({ props }) => {
   const [imageFile, setImageFile] = useState();
   const [imagePreview, setImagePreview] = useState("");
 
+  console.log(styles);
```
remove console.log

kafaa-dev replied:
oh iya hehe

kafaa-dev committed:
Remove console.log

kafaa-dev committed:
Revert "Remove console.log"

This reverts commit 9a594ee140864d6b927ef493e46ff37b97fd23a0.

kafaa-dev committed:
Remove console.log

Saya kira mas @vitorizkiimanda mengubah filenya karena ada tulisan
vitorizkiimanda suggested changes

Makanya commit sebelumnya saya revert supaya saya bisa approve change nya, sepertinya hanya comment
